EnglishTranslation-master/SKILL.tsv

```diff
@@ -2268,13 +2268,13 @@ SKILL_20150317_002267	Decreases the monster's armor with [Aspersion] by 8% per a
 SKILL_20150317_002268	Monstrance: Following Enemies	몬스트런스: 적을 따라다님				
 SKILL_20150317_002269	Your Magic Circle of [Monstrance] will follow nearby monsters.	스킬 [몬스트런스] 마법진이 주변 가까운 몬스터를 따라다닙니다				
 SKILL_20150317_002270	Monstrance: Evasion Defense	몬스트런스: 회피 방어				
-SKILL_20150317_002271	When allies are within [Monstrance] magic circle, evasion and Physical defense of target increase by 2% per attribute level.	스킬 [몬스트런스] 마법진에 아군이 들어가면 대상의 회피, 물리 방어력이 5초 동안 특성 레벨 당 2% 증가합니다.				
+SKILL_20150317_002271	When allies are within [Monstrance] magic circle, evasion and Physical Defense of target increase by 2% per attribute level.	스킬 [몬스트런스] 마법진에 아군이 들어가면 대상의 회피, 물리 방어력이 5초 동안 특성 레벨 당 2% 증가합니다.				
```

Comment:
`Increases the Physical Defense and Evasion of a target by 2% per attribute level when allies are within [Monstrance]'s magic circle.`